### PR TITLE
Profile inline edit

### PR DESCRIPTION
**Pull request by eileenabadi**
profile-edit-page inline edit.

### CODE REVIEW

File: client/src/app/main/profile/edit/profile-edit.component.scss [hunk 1]
```diff
@@ -1,13 +1,9 @@
+@import "../../../../styles/constants";
```
Comment (eileenabadi) [inline client/src/app/main/profile/edit/profile-edit.component.scss]: already gone in the newest bug-on-label

File: client/src/app/main/profile/edit/profile-edit.component.ts [hunk 2]
```diff
@@ -1,4 +1,4 @@
-import { Component, OnInit, ViewChild } from '@angular/core';
+import {Component, OnInit, ViewChild} from '@angular/core';
```
Comment (eileenabadi) [inline client/src/app/main/profile/edit/profile-edit.component.ts]: already corrected in the newest bug-on-label

File: client/src/app/main/profile/edit/profile-edit.component.ts [hunk 3]
```diff
@@ -57,18 +57,11 @@ export class ProfileEditComponent implements OnInit {
   }
 
   public refreshData() {
```
Comment (jpastena) [inline client/src/app/main/profile/edit/profile-edit.component.ts]: this function seems now not necessary (it used only on the init function)

File: client/src/app/main/profile/edit/profile-edit.component.ts [hunk 4]
```diff
@@ -57,18 +57,11 @@ export class ProfileEditComponent implements OnInit {
...
-    this.resetArray();
     this.retrieveLabel();
     this.retrieveProfile();
     this.addLabelFormGroup.get('name').setValue('');
```
Comment (jpastena) [inline client/src/app/main/profile/edit/profile-edit.component.ts]: probably as well this call. (the refresh data is called only on the init)

**Pull request merged**